typescript-client-sdk/client.ts

```diff
@@ -40,10 +40,11 @@ export class HathoraClient {
     return res.data.token;
   }
 
-  public async loginSiwe(message: string, signature: string): Promise<string> {
+  public async loginSiwe(message: string, signature: string, nonceToken: string): Promise<string> {
     const res = await axios.post(`http://localhost:3000/verify`, {
       message,
       signature,
+      nonceToken
     });
```

Comment:
add a header to the axios post called `X-Nonce-Token` and assign it to the nonceToken parameter.